EcosystemSampleApp/SampleAppViewController.swift

```diff
@@ -339,5 +342,61 @@ class SampleAppViewController: UIViewController, UITextFieldDelegate {
             }
         }
     }
+    
+    @IBAction func orderConfirmationTapped(_ sender: Any) {
+        guard let offerId = lastOfferId else{
+            
+            let alert = UIAlertController(title: "No Order has Been Made", message: "No Order was sent in this session yet.", preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "Oh ok", style: .cancel, handler: { [weak alert] action in
+                alert?.dismiss(animated: true, completion: nil)
```

Comment:
@yosriz the cancel will close on its own. you can simply call `alert.addAction(UIAlertAction(title: "Oh ok", style: .cancel))`